lib/ActivityStream.jsm

```diff
@@ -483,17 +483,25 @@ const PREFS_CONFIG = new Map([
           DE: ["de", "de-DE", "de-AT", "de-CH"],
         };
 
+        const dsBasicLayoutMatrix = {
+          DE: ["de", "de-DE", "de-AT", "de-CH"],
+        };
+
         // Verify that the current geo & locale combination is enabled
         const isEnabled =
           !!dsEnablementMatrix[geo] && dsEnablementMatrix[geo].includes(locale);
 
+        const isBasicLayout =
+          !!dsBasicLayoutMatrix[geo] && dsBasicLayoutMatrix[geo].includes(locale);
+
         return JSON.stringify({
           api_key_pref: "extensions.pocket.oAuthConsumerKey",
           collapsible: true,
           enabled: isEnabled,
           show_spocs: showSpocs({ geo }),
           hardcoded_layout: true,
           personalized: true,
+          ...(isBasicLayout ? { hardcoded_basic_layout: true } : {}),
```

Comment:
So, is the idea here to make Germany get the 1 row basic layout by default and then rollout a pref change to switch some of the population to the 7 row layout?

Reply:
Yes, but given this patch was still in review, I think I was able to get my solution into this patch.